forge-game/src/main/java/forge/game/staticability/StaticAbilityContinuous.java

```diff
@@ -330,6 +312,11 @@ public String apply(String input) {
                     String[] restrictions = params.containsKey("SharedRestrictions") ? params.get("SharedRestrictions").split(",") : new String[] {"Card"};
                     addKeywords = CardFactoryUtil.sharedKeywords(addKeywords, restrictions, zones, hostCard, stAb);
                 }
+
+                if (params.containsKey("FromDraftNotes")) {
+                    addKeywords = Lists.newArrayList(hostCard.getController().getDraftNotes().getOrDefault(hostCard.getName(), "").split(","));
```

Comment:
The Previous AddKeywords aren't even used?

And imo I would like it more if the creature(/names) were drafted there, and the Keywords are calculated

Reply:
I'm kinda concerned about the performance consideration of not extracting the keywords out of the PaperCards. If I have to do it in the game, its going to happen every tick.

Reply:
imo we could optimize it later

Reply:
Cool. I'll add it to a futures list.